content/en/_index.html

```diff
@@ -13,30 +13,322 @@
+              Automating signing and verification into a few simple CLI commands
+            </li>
+            <li>
+              Pluggable design allows you to develop plugins and ecosystem
+              integration
+            </li>
+            <li>Provides SDK which allows you to develop your own client</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="feature-card">
+      <div class="order-number is-digit">04</div>
+      <div class="feature-title">
+        <h1 class="text-left">Multi-registry support</h1>
+      </div>
+      <div class="feature-description text-left">
+        <div class="description is-bullets">
+          <ul>
+            <li>
+              It supports push and store signatures alongside the artifacts in
+              OCI registries, such as Docker Hub, ACR, Zot registry, etc.
+            </li>
+            <li>
+              Portable and immutable, you can copy an artifact with its
+              signature across registries
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="feature-card">
+      <div class="order-number is-digit">05</div>
+      <div class="feature-title">
+        <h1 class="text-left">
+          Community- <br />
+          driven
+        </h1>
+      </div>
+      <div class="feature-description text-left">
+        <div class="description is-bullets">
+          <ul>
+            <li>
+              100% open source, built and improved by the active communitye
+            </li>
+            <li>100+ contributors in total, from multiple organizations</li>
+            <li>Fast iteration cadence and open community governance</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+{{% /blocks/lead %}} {{% blocks/lead color="white" %}}
+<h1 class="title">Adopted and trusted by</h1>
+<p class="width-60 is-centered font-size-md">
+  Industry-leading enterprises and organizations are using Notary v2 for
+  research, production, and integration with security products. If you are using
+  Notary v2, please share your case with us
+</p>
 
-  <ul>
-    <li>Sign, store, and verify any OCI artifact, including container images</li>
-    <li>Signing and verifying uses public/private key verification</li>
-    <li>Signing does not change the <code>@digest</code> or <code>artifact:tag</code> reference</li>
-    <li>Signatures are portable across OCI-conformant registries</li>
-    <li>Support for multiple signatures</li>
-    <li>Designed to support multiple development flows</li>
-  </ul>
+<div
+  class="flex gap-md width-80 is-centered font-size-sm margin-md-top is-mobile-responsive fullwidth-tablet"
+>
+  <div class="horizontalCard flex gap-md is-align-items-center">
+    <div class="img-container-md">
+      <img src="aws_logo.svg" alt="Aqua logo" />
+    </div>
+    <div class="text-container text-left">
+      <p>
+        AWS Signer team is using and contributing to Notation, building the
```

Comment:
@vaninrao10 requested that we remove the Signer from the text.

Reply:
@toddysm As Vani mentioned above, this will be addressed in a follow-up PR. I will raise a PR to update the description. Let's agree on merging Uche's change first.

Reply:
I already pushed a fix for this.
I'm trying to fix the issue of commits not being verified.